### PR TITLE
2022 MPPTW in CC landing page

### PR DESCRIPTION
**Pull request by flaviogalizia**
Please review , thanks!

### CODE REVIEW

File: sites/diverseeducation.com/server/templates/static-pages/mppwcc-yearly.marko [hunk 1]
```diff
@@ -302,6 +301,77 @@ $ const cheeGoals = currentYearData ? currentYearData.cheeGoals: title;
...
+            <thead>
+              <tr>
+                <td><b>Institution</b></td>
+                <for|label| of=tableFourLabels>
+                  <td>
+                    <b>${label}</b>
+                  </td>
+                </for>
+              </tr>
+            </thead>
```
Comment (Shinsina) [inline sites/diverseeducation.com/server/templates/static-pages/mppwcc-yearly.marko:322]: Try experimenting with width attributes on the td's here, I believe ```sites/diverseeducation.com/server/templates/static-pages/mppwsa-yearly``` did something similar in order to get the tables to fill out the pages better as here is what I'm getting as of time of comment:


Screen Shot 2022-05-25 at 10 53 14 AM
Screen Shot 2022-05-25 at 10 53 20 AM

**Review by Shinsina — CHANGES_REQUESTED** [review]
Couple of tweaks here but overall this looks pretty good, feel free to reply back on respective comments here or respond back on the associated Trello card.

**Comment by Shinsina** [comment]
[2022 Most Promising Places to Work in Community Colleges](https://trello.com/c/PZHgD8hM/188-2022-most-promising-places-to-work-in-community-colleges)

File: packages/global/components/layouts/content/company.marko 2.js [hunk 2]
```diff
@@ -0,0 +1,247 @@
+// Compiled using marko@4.20.2 - DO NOT EDIT
```
[the remark this answers is not in the document]
Comment (flaviogalizia) [reply packages/global/components/layouts/content/company.marko 2.js:1]: I didn't add the comment on the file, it was there.

File: sites/diverseeducation.com/server/templates/static-pages/mppwcc-yearly.marko [hunk 3]
```diff
@@ -37,7 +37,7 @@ $ const cheeGoals = currentYearData ? currentYearData.cheeGoals: title;
       <@section>
         <div class="row">
           <div class="col">
-            <h1 class="page-wrapper__website-section-name">$!{title}</h1>
+           <div align="center"><h1 class="page-wrapper__website-section-name">$!{title}</h1></div>
```
[the remark this answers is not in the document]
Comment (jwade1327) [reply sites/diverseeducation.com/server/templates/static-pages/mppwcc-yearly.marko:40]: @flaviogalizia @Shinsina  For consistency with the MPPWSA page we should either center both titles or leave as is before your edit

File: sites/diverseeducation.com/server/templates/static-pages/mppwcc-yearly.marko (hunk 1 above)
Comment (jwade1327) [reply sites/diverseeducation.com/server/templates/static-pages/mppwcc-yearly.marko:322]: @flaviogalizia See the previous tables in this file

File: sites/diverseeducation.com/server/templates/static-pages/configs/mppwcc-yearly/2022.js [hunk 4]
```diff
@@ -0,0 +1,484 @@
...
+    },
+    {
+    thumbnail: 'HACC-President-John-Sygielski-Lois-Elfman.jpg',
+    name: 'Harrisburg Area Community College',
+    body: 'This is the fourth such recognition for Harrisburg Area Community College (HACC), founded in 1964. Boasting one of the oldest community colleges in Pennsylvania, Harrisburg serves approximately 17,000 degree-seeking students and more than 8,300 remedial and workforce development students. “HACC embraces this recognition as confirmation of our ongoing commitment to create an atmosphere of diversity, inclusion, and belonging that affirms, celebrates and supports all members of our HACC community,” says Dr. John J. Sygielski, president and CEO.',
+    },
+    {
+    thumbnail: 'Reber-Christopher-Hudson-County-Lois-Elfman.jpg',
```
Comment (jwade1327) [inline sites/diverseeducation.com/server/templates/static-pages/configs/mppwcc-yearly/2022.js:57]: Ending is .jpeg

File: sites/diverseeducation.com/server/templates/static-pages/configs/mppwcc-yearly/2022.js [hunk 5]
```diff
@@ -0,0 +1,484 @@
...
+    },
+    {
+    thumbnail: 'Rowina-Tomaneng-San-Jose-Lois-Elfman.jpg',
+    name: 'San Jose City College',
+    body: 'Founded in 1921, San Jose City College (SJCC) is a public community college in San Jose, Calif. The college offers associate degree programs and certificates; innovative workforce development, which includes several strategic partnerships; online distance learning; and clear pathways to transfer to a four-year institution. “This is recognition of SJCC’s deep commitments to diversity, equity, inclusion and belonging,” says President Dr. Rowena M. Tomaneng. “We are a dual designed minority serving institution (Hispanic serving and Asian American Native American Pacific Islander serving) striving to create inclusive learning and working environments for our students and employees.”',
+    },
+    {
+    thumbnail: 'Bentley-Wallace-South-Arkansas-Lois-Elfman.jpg',
```
Comment (jwade1327) [inline sites/diverseeducation.com/server/templates/static-pages/configs/mppwcc-yearly/2022.js:117]: Ending is .png

File: sites/diverseeducation.com/server/templates/static-pages/configs/mppwcc-yearly/2022.js [hunk 6]
```diff
@@ -0,0 +1,484 @@
...
+    },
+    {
+    thumbnail: 'Gregory-Haile-Broward-Lois-Elfman.jpg',
+    name: 'Broward College',
+    body: 'In Southeast Florida and part of the Florida College System, Broward College is a public institution established in 1959. Under its current name since 2008, the college has three campuses and additional partnerships around Broward County. It serves approximately 63,000 students annually with nearly 5,000 part-time and full-time faculty and staff. “Broward College and the work we do is a vital pathway for advancing people from all backgrounds,” says President Gregory Adam Haile. “Our commitment is to continue to attract the best and brightest professionals eager to improve the lives of our residents through post-secondary education.”',
+    },
+    {
+    thumbnail: 'Misty-Hatfield-Central-Carolina-TC-Lois Elfman.jpg',
```
Comment (jwade1327) [inline sites/diverseeducation.com/server/templates/static-pages/configs/mppwcc-yearly/2022.js:22]: Ending is .JPG

File: sites/diverseeducation.com/server/templates/static-pages/configs/mppwcc-yearly/2022.js [hunk 7]
```diff
@@ -0,0 +1,484 @@
...
+    },
+    {
+    thumbnail: 'Jim-Murdaugh-Tallahassee-Lois-Elfman.jpg',
+    name: 'Tallahassee Community College',
+    body: 'Tallahassee Community College (TCC) was established in 1966. It is part of the Florida College System and serves over 20,000 students. Tallahassee offers pathway programs to several four-year institutions in the area, including Florida State University and Florida A&M University. “It has beenmy vision since I became president to ensure TCC is anemployer of choice,” says Dr. Jim Murdaugh. “We have the best people on our team, who work hard each and every day for our students, so we want to be the best organization we can for them.”',
+    },
+    ],
```
[the remark this answers is not in the document]
Comment (flaviogalizia) [reply sites/diverseeducation.com/server/templates/static-pages/configs/mppwcc-yearly/2022.js:131]: Is there a way to indent doing one click?

[the remark this answers is not in the document]
Comment (Shinsina) [reply sites/diverseeducation.com/server/templates/static-pages/configs/mppwcc-yearly/2022.js:131]: Assuming eslint is working inside of VS Code, you can right click on the error and hit "fix all auto-fixable problems" and then resave the file.
Screen Shot 2022-05-25 at 11 38 11 AM

File: sites/diverseeducation.com/server/templates/static-pages/configs/mppwcc-yearly/2022.js (hunk 6 above)
Comment (Shinsina) [reply sites/diverseeducation.com/server/templates/static-pages/configs/mppwcc-yearly/2022.js:22]: I believe this one may be due to the space between Lois and Elfman.

File: sites/diverseeducation.com/server/templates/static-pages/configs/mppwcc-yearly/2022.js (hunk 7 above)
[the remark this answers is not in the document]
Comment (flaviogalizia) [reply sites/diverseeducation.com/server/templates/static-pages/configs/mppwcc-yearly/2022.js:131]: thanks!

**Review by flaviogalizia — COMMENTED** [review]
- Corrected indentation
- Added CSS class on title section.
- Corrected images with wrong file name.
- Added width on table "Enrollment Type"
- Deleted company.marko2.js file

**Review by jwade1327 — CHANGES_REQUESTED** [review]

File: packages/global/components/layouts/content/company.marko 2.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (jwade1327) [reply packages/global/components/layouts/content/company.marko 2.js:1]: @flaviogalizia This file needs to be deleted per Jake's comment above

**Review by jwade1327 — CHANGES_REQUESTED** [review]

**Review by Shinsina — APPROVED** [review]

**Pull request merged**